### DISCUSSION TITLE
Create theme class

### DISCUSSION THREAD

**Discussion by joselrodriguezd** [Power-Components/livewire-powergrid#564] [General]
hi, maybe this error is easy to fix, but when i try to create a theme i get this error: PowergridThemes\ThemeBase' not found how i cant fixes, and another question, how i can show all the row result instead of 10.

thank you

**Comment by luanfreitasdev** [comment]
Hello. can you show the theme you created? what is the framework?

**Reply by joselrodriguezd** [reply]
Hi, thanx from reply, i follow this instructions https://livewire-powergrid.com/#/custom-theme, i working in laravel 8, tailwind 3, powergrid 2.4.

**Reply by joselrodriguezd** [reply]
please help me when you can with this concern

**Reply by luanfreitasdev** [reply]
Did you see this demo? there are some examples you can check out 

https://github.com/Power-Components/powergrid-demo-misc

**Reply by joselrodriguezd** [reply]
thanks u.

**Reply by joselrodriguezd** [reply]
Hi, i follow the examples from the demo, In my localhost works fine, but when i upload to the server in production, give a error:

Class "App\Http\Livewire\PowergridThemes\Tailwind" not found (View: /workspace/resources/views/livewire/prestamos/valores.blade.php)

![2022-08-18_14h07_28](https://user-images.githubusercontent.com/48702587/185475051-b386b477-6dea-4687-91c9-15e47df696e8.png)
![2022-08-18_14h07_56](https://user-images.githubusercontent.com/48702587/185475053-62a7cd2c-e5a2-4f33-b964-d0b898e99db1.png)
![2022-08-18_14h08_13](https://user-images.githubusercontent.com/48702587/185475054-351e0085-16dc-4393-ab51-6d81efede351.png)
![2022-08-18_14h08_29](https://user-images.githubusercontent.com/48702587/185475056-ba17b4d0-32fa-4e2f-85cc-e0efe0f7fc21.png)
